samples/ribbon/ribbondemo.cpp

```diff
@@ -368,8 +368,8 @@ MyFrame::MyFrame()
         sizer_panelcombo2->SetMinSize(wxSize(150, -1));
 
         wxRibbonButtonBar* bar = new wxRibbonButtonBar(sizer_panel, wxID_ANY);
-        bar->AddButton(ID_BUTTON_XX, "xx", ribbon_xpm);
-        bar->AddButton(ID_BUTTON_XY, "xy", ribbon_xpm);
+        bar->AddButton(ID_BUTTON_XX, "xx", wxBitmap(ribbon_xpm));
```

Comment:
This doesn't need to be done in this PR, of course, but it would be nice if we could have a high resolution version of this (and all the other) XPMs or, ideally, an SVG.

Reply:
I just updated the example to make it work. There are also some other parts of wxRibbonBar where such updates are necessary, for example for drop down arrows... I would rather do it later.